stock_putaway_same_location/tests/test_same_location_putaway_strategy.py

```diff
@@ -0,0 +1,277 @@
+# Copyright (C) 2018 by Camptocamp
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+from datetime import datetime
+from odoo.tests import common
+
+
+class SameLocationPutawayCase(common.SavepointCase):
+    @classmethod
+    def setUpClass(cls):
+        super().setUpClass()
+
+        # Common data
+        cls.location_stock = cls.env.ref('stock.stock_location_stock')
+        tested_putaway_strategy = cls.env['product.putaway'].create({
+            'name': 'Tested putaway strategy',
+            'method': 'previous/empty',
```

Comment:
this crashes in Travis:

ValueError: Wrong value for product.putaway.method: 'previous/empty'